tests/integration/components/impagination-dataset-test.js

```diff
@@ -96,6 +96,17 @@ describe('Integration | Component | ImpaginationDataset', function() {
       expect(first.content).to.be.null;
     });
 
+    describe("when the dataset is recomputed", function() {
+      beforeEach(function() {
+        init.reset();
```

Comment:
Yes. It clears out any call data from the spy, that way subsequent assertions can only consider calls made to the spy _after_ the call to `reset()`